extractor/src/main/java/org/schabi/newpipe/extractor/services/youtube/ItagItem.java

```diff
@@ -150,6 +162,7 @@ public MediaFormat getMediaFormat() {
 
     // Audio fields
     public int avgBitrate = -1;
+    public int sampleRate = -1;
```

Comment:
There should be either a JDoc comment for the "unknown" status or a constant (same for `fps`).